code/src/store/slices/localCart.ts

```diff
@@ -0,0 +1,76 @@
+import { PayloadAction, createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+
+import { initialCart } from "@/constants/cart";
+import { useAppSelector } from "@/hooks/use-redux/useRedux";
+import cartApi from "@/store/api/cartApi";
+import { sliceNames } from "@/store/constants";
+import { RootState } from "@/store/store";
+import { CartItem } from "@/types/cart.types";
+import getLocalStorageCart from "@/utils/get-local-storage-cart/getLocalStorageCart";
+
+const initialState = getLocalStorageCart();
+
+export const removeFromLocalCart = createAsyncThunk(
+  "localCart/removeFromCart",
```

Comment:
maybe would be better to move it to constants

Reply:
Thanks, I replaced it with the same approach used in userSlice:  `${sliceNames.localCart}/removeFromCart`